app/src/services/lifecycle.js

```diff
@@ -33,4 +33,39 @@ module.exports = {
 
     return lifecycleHistory;
   },
+
+  /**
+   * @function getLatestEnvAction
+   * Finds the latest lifecycle action for `env`
+   * @param {string} acronymId The desired acronymId
+   * @param {string} env The desired environment
+   * @returns {object} A lifecycle object if it exists
+   */
+  async getLatestEnvAction(acronymId, env) {
+    const lifecycle = await db.Lifecycle.findAll({
+      where: {
+        acronymId: acronymId,
+        env: env
+      },
+      order: [
+        ['updatedAt', 'DESC']
```

Comment:
This will work I think (since in this case updatedAt will always be the same), but createdAt would be the relevant field here if I'm not mistaken?

Reply:
I guess so. especially if a password reset would match a record.
i can change that now to created.

Reply:
yeah i assumed either was good. we're calling it 'environment last updated' in the ui.